services/keyword_service/keyword.go

```diff
@@ -0,0 +1,100 @@
+package keyword_service
+
+import (
+	"encoding/csv"
+	"errors"
+	"io"
+	"mime/multipart"
+	"os"
+	"path/filepath"
+
+	"github.com/gin-gonic/gin"
+	errorHandler "github.com/gutakk/go-google-scraper/helpers/error_handler"
+	"github.com/gutakk/go-google-scraper/models"
+)
+
+const (
+	fileFormatError         = "file must be CSV format"
+	fileLengthError         = "CSV file must contain between 1 to 1000 keywords"
+	invalidDataError        = "invalid data"
+	somethingWentWrongError = "something went wrong, please try again"
+)
+
+type Keyword struct {
```

Comment:
Should this struct be called `Keyword` as it's a bit confusing with the `Keyword` model struct 🤔 This struct is for KeywordService so it could be named after it?

Reply:
Yes I think it is confusing too but I cannot think of the good name. Thanks for pointing this out
Fixed in e60715e61314131376aa6ffc43039aa45388884d